doc/userguide/source-verification.rst

```diff
@@ -0,0 +1,83 @@
+
+Verifying Source Tarball
+========================
+
+Once the Suricata release tarball has been downloaded, the PGP signature should
+be verified. This can be done using the GPG application and is usually
+available on Linux/BSD systems without having to manually install any
+additional packages. For Mac or Windows systems installation packages can be
+found at `<https://gnupg.org/>`_.
+
+Verification Steps
+------------------
+
+These verification steps are for general guidance, the exact process and
+commands may vary between operating systems.
+
+Downloading the sig File
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The sig file needs to be downloaded as well as the tarball. Both files can be
+found at `<https://suricata.io/download/>`_.
+
+Importing the OISF Signing Key
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once both the sig file and suricata tarball files are obtained, the OISF
+signing key should be imported to the local gpg keyring. This can be done by
+running the following command::
+
+  $ gpg --receive-keys 2BA9C98CCDF1E93A
+
+The above command should produce output similar to the following::
+
+  gpg: key 2BA9C98CCDF1E93A: public key "Open Information Security Foundation
+  (OISF) <releases@openinfosecfoundation.org>" imported
+  gpg: Total number processed: 1
+  gpg:               imported: 1
+
+Verifying the Suricata Tarball
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To verify the contents of the Suricata tarball the following command should be
+ran::
+
+  $ gpg --verify suricata-7.0.5.tar.gz.sig suricata-7.0.5.tar.gz
```

Comment:
Considering how literal some interpretations can be, I think it's worth mentioning that the proper Suricata version should match with what they have installed, in their command-run. 

Or maybe rephrase to indicate "To verify the Suricata-7.0.5 distribution archive, for instance, you could run"